tests/test_optimizer.py

```diff
@@ -561,25 +562,29 @@ def test_wrappers():
 @pytest.mark.slow
 def test_all_optimizers():
     """Just tests that the optimizers run without error, eg tests for the wrappers."""
-    eq = desc.examples.get("SOLOVEV")
-    fobj = ObjectiveFunction(ForceBalance(eq=eq))
-    eobj = ObjectiveFunction(Energy(eq=eq))
+    eqf = desc.examples.get("SOLOVEV")
+    eqe = eqf.copy()
+    eqe._node_pattern = "quad"
```

Comment:
Seems like this is because node pattern was not an ioable attribute at the time SOLOVEV was solved and saved. Might be good to add a todo marker to remove this when solved in case solovev node pattern is changed in future.

Reply:
No, it is saved, but the example uses a `jacobi` pattern with force balance as the objective. This creates another version that uses `quad` to avoid the warning when creating the energy objective.